safe/report/extractors/analysis_detail.py

```diff
@@ -361,6 +361,12 @@ def sort_classes(_row):
                 row = details[row_idx]
                 row = row[:column_index] + row[column_index + 1:]
                 details[row_idx] = row
+            # reduce total affected and not affected column index by 1
+            # since we are removing a column
+            if group_key == 'affected':
```

Comment:
Do we have this `affected` group in the definitions module? If so, it's better to get from the definiton.